src/Symfony/Component/Validator/Tests/ValidatorBuilderTest.php

```diff
@@ -81,7 +81,7 @@ public function testAddMethodMappings()
      */
     public function testEnableAnnotationMappingWithDefaultDoctrineAnnotationReader()
     {
-        $this->assertSame($this->builder, $this->builder->enableAnnotationMapping());
+        $this->assertSame($this->builder, $this->builder->enableAttributeMapping());
```

Comment:
this test is really about annotation mapping. So I would rather expect both deprecations in it than changing method